int32 EFFECT_SOUND = 2
int32 MIDI = 3
int32 mode
int32 effect_id
int32 volume
toio_pkg/Note[] notes